### ISSUE TITLE
Test on N3DS

### ISSUE THREAD

**Issue by Arcterus** [Arcterus/corbenik#5]
People seem to indicate that corbenik isn't working in the original repo on B9S, but everything seems okay on my O3DS using B9S.  Need to check whether it's a N3DS-only issue or if I'm missing something.

**Comment by Derek1199** [comment]
Yeah, it's not working. TWL And AGB Firm patching fail to patch and the screen is just white in the menu so you can't see anything.

**Comment by Arcterus** [comment]
Ah, thanks for confirming.  I'll have to figure out a way to test on a N3DS then...
